villa-villa_diana/script.js

```diff
@@ -0,0 +1,51 @@
+let nameField = document.getElementById("name");
+let commentField = document.getElementById("user_comment");
+
+function newComment() {
+  if (nameField.value.length > 0 && commentField.value.length > 0) {
```

Comment:
can use guard clause 
```suggestion
  if (nameField.value.length && commentField.value.length) {
```